.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
```

Comment:
IMO, this kind of file should not be added to your PR.

Reply:
TBH, I don't know why this is included. All I did, is rebased my branch with the current main branch.

Reply:
Could you please rebase yours with the last upstream `main` branch?

Reply:
I did but seems to be no changes to this issue. What's weird, is when I am doing a `git diff upstream/main` locally, I cannot see that these files are changed.